src/discovery/kubernetes/Akka.Discovery.KubernetesApi/README.md

```diff
@@ -1,18 +1,49 @@
 # Kubernetes Api Discovery
 
-The Kubernetes API can be used to discover peers and form an Akka Cluster. The `KubernetesApi` 
-mechanism queries the Kubernetes API server to find pods with a given label. A Kubernetes service 
-isn’t required for the cluster bootstrap but may be used for external access to the application.
+The Kubernetes API can be used to discover peers and form an Akka Cluster. The `KubernetesApi` mechanism queries the Kubernetes API server to find pods with a given label. A Kubernetes service isn’t required for the cluster bootstrap but may be used for external access to the application.
```

Comment:
Is what you mean here: "Cluster.Boostrap queries the ports on the pods directly and doesn't need a Kubernetes `Service` to expose them." ?